BlazarUI/app/stylus/components/module-modal.styl

```diff
@@ -0,0 +1,12 @@
+.module-modal--container
+  height 200px
+
+.module-modal--modules
+  float left
+
+.module-modal--downstream
+  float right
+
+.module-modal--title
+  font-size 1.5em
```

Comment:
same as ^, i would either use `h` tags, or if you want to have custom modal styles, I would make this more general for all modals, eg `modal-title`, so you're not writing custom css for every different modal that exists.